README.md

```diff
@@ -136,6 +136,7 @@
 - [vuepress-theme-macgs](https://github.com/EasyHexo/vuepress-theme-macgs) - :dizzy: A vuepress theme for [easyhexo.com](https://easyhexo.com)
 - [vuepress-theme-jupiter](https://github.com/zlliang/vuepress-theme-jupiter) - A simple theme using [Bulma-jupiter](https://github.com/zlliang/bulma-jupiter) styles, and supporting LaTeX rendering
 - [vuepress-theme-simple](https://github.com/viko16/vuepress-theme-simple) - ✏️ Very Simple Blog Theme for VuePress
+- [vuepress-theme-simple-blog](https://github.com/g-plane/vuepress-theme-simple-blog) - Forked from "vuepress-theme-simple", but provides better support for [@vuepress/plugin-blog](https://github.com/ulivz/vuepress-plugin-blog).
```

Comment:
Are you sure that a theme provides better support for [@vuepress/plugin-blog](https://github.com/ulivz/vuepress-plugin-blog)?

Reply:
You can compare the code if you want.